LongReadRNABenchmark/LongReadRNABenchmarkTasks.wdl

```diff
@@ -0,0 +1,697 @@
+        --gtf ~{bambuGTF} \
+        --tool "bambu" \
+        --output "~{datasetName}_bambu_reduced_db"
+
+        /usr/local/src/IsoQuant-3.1.1/misc/reduced_db_gffcompare.py \
+        --genedb ~{reducedAnnotationPrefix} \
+        --gtf ~{flairGTF} \
+        --tool "flair" \
+        --output "~{datasetName}_flair_reduced_db"
+
+        /usr/local/src/IsoQuant-3.1.1/misc/reduced_db_gffcompare.py \
+        --genedb ~{reducedAnnotationPrefix} \
+        --gtf ~{talonGTF} \
+        --tool "talon" \
+        --output "~{datasetName}_talon_reduced_db"
+
+        tar -zcvf ~{datasetName}_isoquant_reduced_db.tar.gz ~{datasetName}_isoquant_reduced_db
+        tar -zcvf ~{datasetName}_stringtie_reduced_db.tar.gz ~{datasetName}_stringtie_reduced_db
+        tar -zcvf ~{datasetName}_bambu_reduced_db.tar.gz ~{datasetName}_bambu_reduced_db
+        tar -zcvf ~{datasetName}_flair_reduced_db.tar.gz ~{datasetName}_flair_reduced_db
+        tar -zcvf ~{datasetName}_talon_reduced_db.tar.gz ~{datasetName}_talon_reduced_db
+    >>>
+
+    output {
+        File gffCompareOutputIsoQuant = "~{datasetName}_isoquant_reduced_db.tar.gz"
+        File gffCompareOutputStringTie = "~{datasetName}_stringtie_reduced_db.tar.gz"
+        File gffCompareOutputBambu = "~{datasetName}_bambu_reduced_db.tar.gz"
+        File gffCompareOutputFlair = "~{datasetName}_flair_reduced_db.tar.gz"
+        File gffCompareOutputTalon = "~{datasetName}_talon_reduced_db.tar.gz"
+        File monitoringLog = "monitoring.log"
+    }
+
+    runtime {
+        docker: docker
+        cpu: cpu
+        memory: "~{memory} GiB"
+        disks: "local-disk ~{diskSizeGB} HDD"
+    }
+}
+
+task DenovoAnnotationGFFCompare {
+    input {
+        File isoQuantGTF
+        File stringTieGTF
+        File bambuGTF
+        File bambuGTFCounts
+        File flairGTF
+        File talonGTF
+        String datasetName
+    }
+
+    String docker = "us.gcr.io/broad-dsde-methods/kockan/isoquant-gffcompare:latest"
+    Int cpu = 8
+    Int memory = 64
+    Int diskSizeGB = 300
+    File monitoringScript = "gs://broad-dsde-methods-tbrookin/cromwell_monitoring_script2.sh"
+
+    String isoQuantBasename = basename(isoQuantGTF)
+    String stringTieBasename = basename(stringTieGTF)
+    String bambuBasename = basename(bambuGTF)
+    String flairBasename = basename(flairGTF)
+    String talonBasename = basename(talonGTF)
+
+    command <<<
+        bash ~{monitoringScript} > monitoring.log &
+
+        mv ~{isoQuantGTF} .
+        mv ~{stringTieGTF} .
+        mv ~{bambuGTF} .
+        mv ~{bambuGTFCounts} .
+        mv ~{flairGTF} .
+        mv ~{talonGTF} .
+
+        echo "~{isoQuantBasename} isoquant" > gtfs.list
+        echo "~{stringTieBasename} stringtie" >> gtfs.list
+        echo "~{bambuBasename} bambu" >> gtfs.list
+        echo "~{flairBasename} flair" >> gtfs.list
+        echo "~{talonBasename} talon" >> gtfs.list
+
+        /usr/local/src/IsoQuant-3.1.1/misc/denovo_model_stats.py \
```

Comment:
I think this is basically just a wrapper around gffcompare (with an additional split function).  Wondering if there is a way to remove the wrapper and just make this a call to gffcompare.

Reply:
gffcompare logic now decoupled from IsoQuant. Resolved.